libcaf_io/biicode.conf

```diff
@@ -0,0 +1,44 @@
+# Biicode configuration file
+
+[requirements]
+	 biicode/cmake: 3
+	 outro56/libcaf_core: 1
```

Comment:
building libcaf_io requires libcaf_core
currently this is pointed to my published feed,